_studio/mfx_lib/shared/src/mfx_h264_enc_common_hw.cpp

```diff
@@ -5389,7 +5397,7 @@ void MfxHwH264Encode::SetDefaults(
     if (par.mfx.RateControlMethod == MFX_RATECONTROL_AVBR)
     {
         if (par.mfx.Accuracy == 0)
-            par.mfx.Accuracy = AVBR_ACCURACY_MAX;
+            par.mfx.Accuracy = 100;
```

Comment:
Does AVBR have sense with 100% accuracy?

Reply:
Yes. This accuracy corresponds to "target_percentage" in VAEncMiscParameterRateControl. Its valid range is [1, 100]. As to the real control result, it depends on AVBR algorithm and implementation in lower level. But its valid range is [1, 100].